docs/about/history.rst

```diff
@@ -64,6 +64,7 @@ Fixed
    -  Fix *Allow Null Light* option on *Sim Settings Shadows* not working.
    -  Fix ocean tiles not reverting to *Ocean Renderer > Material* if *Water Body > Override Material* was used and *Water Body* was disabled or removed.
    -  Fix underwater rendering when the camera's culling mask excludes the *Ocean Renderer > Layer*.
+   -  Fix visible "rings" from fast moving dynamic wave sim inputs. Simulation frequency can be increased to improve result further, at the cost of more simulation steps per frame.
```

Comment:
- needs a new line after period for style
- "dynamic wave sim inputs" shouldn't this only be for sphere water interaction?
- an entry for fixing the shader being moved which fixes builds

Reply:
Done